gvm/protocols/gmpv7.py

```diff
@@ -2182,17 +2182,30 @@ def get_credentials(self, *, filter=None, filter_id=None, scanners=None,
 
         return self._send_xml_command(cmd)
 
-    def get_credential(self, credential_id):
+    def get_credential(self, credential_id, credential_format=None):
```

Comment:
Would be nice to require credentials_format as keyword too
```suggestion
    def get_credential(self, credential_id, *, credential_format=None):
```